Source/WebCore/Modules/indexeddb/server/MemoryObjectStore.cpp

```diff
@@ -313,12 +299,10 @@ void MemoryObjectStore::addRecordWithoutUpdatingIndexes(MemoryBackingStoreTransa
 
     ASSERT(m_writeTransaction && m_writeTransaction->isAborting());
 
-    if (!m_keyValueStore) {
-        m_keyValueStore = makeUnique<KeyValueMap>();
+    if (m_orderedKeys)
```

Comment:
This should be `!m_orderedKeys` (tests might fail for this)

Reply:
I think you spotted it. Thank you!